languages/elixir/exercises/concept/list-comprehensions/.docs/introduction.md

```diff
@@ -23,7 +24,7 @@ end
 # => [:b, :d]
 ```
 
-A _cartesian product_ can be created using multiple generators:
+A _cartesian product_ can be created using multiple generators. That means that each value generated by the first generator will be paired once with each value generated by the second generator:
```

Comment:
"Cartesian product" is a mathematical term that many programmers won't know. I had to look it up too... if we want to use it, we need to explain immediately what it means in a simplified way.